nucleus/model.py

```diff
@@ -121,6 +123,7 @@ def from_json(cls, payload: dict, client):
             name=payload["name"],
             reference_id=payload["ref_id"],
             metadata=payload["metadata"] or None,
+            bundle_id=payload["bundle_id"] or None,
```

Comment:
I am curious, is `payload["smth"] or None` is justified, or `payload.get("smth", None)` would be better? Or wee need the code to break if bundle_id is not provided?

Reply:
hmm - interesting question, i think metadata is also not a required field but the thing that calls this defaults it to None also so this is really just a fallback

Reply:
+1 for payload.get("smth", None)